client/src/components/new-request/search.vue

```diff
@@ -349,7 +423,17 @@ export default class NewSearch extends Vue {
 }
 .search-name-btn {
   font-size: 1rem !important;
-  min-height: 45px;
+  min-height: 45px !important;
```

Comment:
I think we're going with 40px tall buttons.
This is one of those special cases, isn't it?

Reply:
lol no no, i think this was done before your clean up. I will fix this up!

Reply:
OK, you can probably inherit most of the styles (unless your spec calls for something different).

Reply:
Copy that, will remove what i can!

Reply:
Oops yea, just checked. This IS a special case, specified at 45px...
A lot of 'special' cases 😅